assets/performanceprofile/scripts/set-rps-mask.sh

```diff
@@ -33,6 +33,15 @@ function find_dev_dir {
 [ -d "${dev_dir}" ] || { sleep 5; find_dev_dir; }  # search failed, wait a little and try again
 [ -d "${dev_dir}" ] || { echo "${dev_dir}" directory not found >&2 ; exit 0; } # the interface disappeared, not an error
 
+net_driver=$(udevadm info -q property "/sys/class/net/${dev}" | grep ID_NET_DRIVER | awk -F '=' '{print$2}')
```

Comment:
I eval is the expected pattern, that is why udev output is formatted like it is.

Reply:
great, so I'm all for `eval` (in this very specific and limited context!) then!

Reply:
Right! I tried with `source` but this works for files. `eval` can do the trick

Reply:
Btw, I think we can use "veth*" prefix to identify the NICs instead of querying udev. At least on my system /sys/class/net/veth1beec93f/uevent does not mention type (some other devices like bridges do) so I assume it is not important.

Reply:
On the cluster's nodes we don't have the "veth*" prefix.
But please refer to https://github.com/openshift/cluster-node-tuning-operator/pull/527#discussion_r1045519342